runner/src/server/lib/consoleMessages/welcome.ts

```diff
@@ -0,0 +1,40 @@
+import config from "server/config";
+
+function welcome() {
+  let message = "";
+  if (!config.isTest) {
+    message += `
+**********
+**********
+Server is running in production mode.
+-- env=${config.env.toString()}
+-- enforceCsrf=${config.enforceCsrf.toString()}
+-- previewMode=${config.previewMode.toString()}
+**********
+**********`;
+  } else {
+    message += `
+**********
+**********
+WARNING: Server is running in insecure development/test mode. 
```

Comment:
if it's a warning it should be console.warning, otherwise logging is being misused